src/filesinkhelper.ipp

```diff
@@ -85,9 +85,9 @@ namespace g3 {
          return ss_entry.str();
       }
 
-      std::string createLogFileName(const std::string &verified_prefix) {
+      std::string createLogFileName(const std::string &verified_prefix, const std::string &logger_id=".g3log.") {
          std::stringstream oss_name;
-         oss_name << verified_prefix << ".g3log.";
+         oss_name << verified_prefix << logger_id;
```

Comment:
I just tried to pull down the latest and built it as follows


git checkout master
git pull upstream master (to get the latest: upstream points to my g3log
repo)
mkdir build
cd build
cmake -DADD_G3LOG_UNIT_TEST=ON ..
make -j

it built fine.

*Contact Information*
*Kjell Hedström*
*Mobile:  +1 - 617.378.1915*

*E-mail: Hedstrom@KjellKod.cc*




On Tue, Jan 26, 2016 at 3:27 PM, jkhoogland <notifications@github.com>
wrote:

> In src/filesinkhelper.ipp
> <https://github.com/KjellKod/g3log/pull/76#discussion_r50911368>:
>
> >           std::stringstream oss_name;
> > -         oss_name << verified_prefix << ".g3log.";
> > +         oss_name << verified_prefix << logger_id;
>
> i ll give it a try,
>
> the current branch doesnt build the tests properly on OSX
> It fails with
>
> [ 55%] Built target g3logger_shared
> make[2]: *** No rule to make target `CMakeFiles/gtest_170_lib.dir/build'.  Stop.
> make[1]: *** [CMakeFiles/gtest_170_lib.dir/all] Error 2
> make: *** [all] Error 2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KjellKod/g3log/pull/76/files#r50911368>.
>